pkg/resources/aws/aws.go

```diff
@@ -324,56 +321,6 @@ func DeleteInstance(cloud fi.Cloud, t *resources.Resource) error {
 	return nil
 }
 
-func DeleteCloudFormationStack(cloud fi.Cloud, t *resources.Resource) error {
```

Comment:
I wonder if we should keep these so that clusters that were once used with cloudformation and later switched to direct will have their CF resources cleaned up when the cluster is deleted

Reply:
This is dead code. It wouldn't help anything to keep it around.